src/lucky_migrator/tasks/db/redo.cr

```diff
@@ -0,0 +1,14 @@
+require "colorize"
+
+class Db::Redo < LuckyCli::Task
+  banner "Rollback then run the last migration"
+
+  def call
+    begin
+      Db::Rollback.new.call
+      Db::Migrate.new.call
+    rescue e : PQ::PQError
```

Comment:
You can remove the begin/rescue because they are handled in those tasks already: https://github.com/luckyframework/lucky_migrator/blob/master/src/lucky_migrator/tasks/db/migrate.cr#L9-L11

Reply:
Ah yes I forgot about that